Lab2_NaiveBayes_Solutions.md

```diff
@@ -54,67 +56,36 @@ Answer the questions below given the sets above.
+5. Imagine you’re working for Reddit. Toxic comments are a real problem, so you decide to build a sentiment classifier using Naive Bayes and an automoderator that deletes negative comments. Your classifier trains on the same training set from Part 1:
 
 **Training Set**
 
     - the dangerous bears
     - bears destroyed the yard
-    + beautiful redwood trees
-
-   The week after training, the Big Game (between Stanford and Cal) happens. Users on /r/bayarea are excited. Alice, a Cal student, wants to support her team and comments `“Go bears!”`. Bob, a Stanford student, wants to support his team and comments `“Go trees!”`. Answer the following questions:
+    + the beautiful redwood trees
 
+   The week after training, the Big Game (between Stanford and Cal) happens. Users on /r/bayarea are excited. Alice, a Cal student, wants to support her team and comments `“Go bears!”`. Bob, a Stanford student, wants to support his team and comments `“Go trees!”`. Answer the following questions. Please keep your responses to parts (a) through (c) concise, while we encourage more in-depth discussions for parts (d) and (e).
+    
    <ol type="a">
-      <li>Is Alice’s comment deleted by your automoderator?
-         <code>Yes -- the presence of "bears" means it is classified as negative.</code>
-      </li>
-      <li>Is Bob’s comment deleted by your automoderator?
-         <code>No -- the presence of "trees" means it is classified as positive.</code>
-      </li>
-      <li>Which two groups of people is your automoderator treating differently, even though membership in these groups doesn’t have a clear connection to comment toxicity?
-         <code>Cal students (people who use "bears") and Stanford students (people who use "trees") are being treated differently.</code>
-      </li>
-      <li>What kind of harm from the lecture is experienced by the group whose comments are getting deleted?
-          <code>Censorship harms (incorrectly deleting non-toxic sentences that simply
-         mention identities).</code>
-      </li>
-      <li>Having realized the harm from part (d) that your automoderator is causing, you change it so that negative comments are not deleted. Instead, they are just highlighted in red and marked “negative”. Now, what kind of harm from lecture is the aforementioned group experiencing?
+       <li>Whose comment, Alice's or Bob's, is deleted by your automoderator, and what kind of harm (from the lecture) are they experiencing?</li>
+           <code>Alice's gets deleted -- the presence of "bears" means it is classified as negative. Bob's does not get deleted as the presence of "trees" means it is classified as positive. Censorship harms (incorrectly deleting non-toxic sentences that simply mention identities).</code>
```

Comment:
Fill out the last sentence a bit more. Censorship is harming Alice, bears, etc. because they are being included in negative hateful comments that they did not participate or influence.